lib/ueberauth/strategy/facebook.ex

```diff
@@ -56,6 +53,22 @@ defmodule Ueberauth.Strategy.Facebook do
       fetch_user(conn, client)
     end
   end
+  def handle_callback!(token) when is_binary(token) do
+    {_module, config} = Application.get_env(:ueberauth, Ueberauth)[:providers][:facebook]
+    token = OAuth2.AccessToken.new(token)
+    client = Ueberauth.Strategy.Facebook.OAuth.client([token: token])
+
+    case OAuth2.Client.get(client, "/me?fields=#{config[:profile_fields]}") do
```

Comment:
@hassox there is not dependency with `conn`. I don't need to attach to `Plug.Conn` for do the workflow